### PR TITLE
Diagnostics module fix (#9987)

### LINKED ISSUE #9987: error page from "OrchardCore.Diagnostics" module isn't presented when [ValidateAntiForgeryToken] attribute is used and validation failed
### Describe the bug
Sometimes, when my "**orchantiforgery_**" cookie expires, and when login page is still opened in browser, I can't log in, because of **[AntiforgeryValidationException](https://github.com/dotnet/aspnetcore/blob/main/src/Antiforgery/src/Internal/DefaultAntiforgery.cs)** which is expected behavior, but instead of seeing error page from 'OrchardCore.Diagnostics' I see default one.

### You can reproduce the issue by removing "**orchantiforgery_**" cookie, or by manipulating with html code.
1. Make sure 'OrchardCore.Diagnostics' module is enabled,
2. Go to 'https://try.orchardcore.net/Login',
3. Inspect page and change value of '__RequestVerificationToken' input,
4. Click on 'Log in',
5. See default 400 error page.
image

### Expected behavior
This error page should be presented to the user:
image

### PR DESCRIPTION
**Pull request by lampersky**
fixes #9987

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Diagnostics/Controllers/DiagnosticsController.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 namespace OrchardCore.Diagnostics.Controllers
 {
+    [IgnoreAntiforgeryToken]
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Diagnostics/Controllers/DiagnosticsController.cs:7]: Can it be added to the method instead?

File: src/OrchardCore.Modules/OrchardCore.Diagnostics/OrchardCore.Diagnostics.csproj [hunk 2]
```diff
@@ -19,4 +19,8 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Module.Targets\OrchardCore.Module.Targets.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" Version="2.2.0" />
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Diagnostics/OrchardCore.Diagnostics.csproj:23]: Is that necessary?

Comment (lampersky) [reply src/OrchardCore.Modules/OrchardCore.Diagnostics/OrchardCore.Diagnostics.csproj]: good point, it's not, I will remove it

File: src/OrchardCore.Modules/OrchardCore.Diagnostics/Controllers/DiagnosticsController.cs (hunk 1 above)
Comment (lampersky) [reply src/OrchardCore.Modules/OrchardCore.Diagnostics/Controllers/DiagnosticsController.cs]: sure, I will move it there

**Review by hishamco — APPROVED** [review]

**Pull request merged**